x-pack/plugins/endpoint/public/applications/endpoint/store/alerts/selectors.ts

```diff
@@ -4,6 +4,68 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import qs from 'querystring';
 import { AlertListState } from '../../types';
 
+/**
+ * Returns the Alert Data array from state
+ */
 export const alertListData = (state: AlertListState) => state.alerts;
+
+/**
+ * Returns the alert list pagination data from state
+ */
+export const alertListPagination = (state: AlertListState) => {
+  return {
+    pageIndex: state.request_page_index,
+    pageSize: state.request_page_size,
+    resultFromIndex: state.result_from_index,
+    total: state.total,
+  };
+};
+
+/**
+ * Returns a boolean based on whether or not the user is on the alerts page
+ */
+export const isOnAlertPage = (state: AlertListState): boolean => {
+  return state.location ? state.location.pathname.endsWith('/alerts') : false;
```

Comment:
(optional I would suggest you do a strict `===` as in `state.location.pathname === '/alerts'`